libopeniscsiusr/idbm.h

```diff
@@ -220,6 +220,7 @@ struct iscsi_session_idbm {
 	char					boot_root[BOOT_NAME_MAXLEN];
 	char					boot_nic[BOOT_NAME_MAXLEN];
 	char					boot_target[BOOT_NAME_MAXLEN];
+	int64_t                                 reopen_log_freq;
```

Comment:
Do you really need 64 bits here for a number that will generally fairly small?

Reply:
Yes, this needs int64_t as _idbm_node_rec_link() in libopeniscsiusr/idbm.c calls _rec_int64() which needs int64_t.